crates/bevy_ecs/src/system/combinator.rs

```diff
@@ -214,7 +214,7 @@ where
     #[inline]
     unsafe fn validate_param_unsafe(&mut self, world: UnsafeWorldCell) -> bool {
         // SAFETY: Delegate to other `System` implementations.
-        unsafe { self.a.validate_param_unsafe(world) && self.b.validate_param_unsafe(world) }
+        unsafe { self.a.validate_param_unsafe(world) }
```

Comment:
Why is this ok? Surely we should check that both params are valid?

Reply:
Ah I should have read above in the PR discussion, can you just add a to-do or at least a comment explanation why we've decided to do this?